library/std/src/sys/xous/stdio.rs

```diff
@@ -0,0 +1,131 @@
+use crate::io;
+
+pub struct Stdin;
+pub struct Stdout {}
+pub struct Stderr;
+
+use crate::os::xous::ffi::{lend, try_lend, try_scalar, Connection};
+use crate::os::xous::services::{log_server, try_connect, LogScalar};
+
+impl Stdin {
+    pub const fn new() -> Stdin {
+        Stdin
+    }
+}
+
+impl io::Read for Stdin {
+    fn read(&mut self, _buf: &mut [u8]) -> io::Result<usize> {
+        Ok(0)
```

Comment:
Xous has no stdin?

Reply:
Not at this time. It will someday, but for now it's just a stub. Its primary interface is through a graphical chat program. It may make sense to add stdin in the future as something that's hooked up to some GUI library, but right now all of this stdout is going to a logging server which is doing the printing.